src/winetricks

```diff
@@ -5094,15 +5160,16 @@ winetricks_init()
 
     # System-specific variables
     case "$W_PLATFORM" in
-        windows_cmd)
+        windows_cygwin)
             WINE=""
             WINE64=""
             WINE_ARCH=""
             WINE_MULTI=""
             WINESERVER=""
             W_DRIVE_C="C:/"
             ;;
-        *)
+        windows_windows) w_die "Platform '$W_PLATFORM' is not implemented in winetricks_init()" ;;
```

Comment:
Not needed.

Reply:
Same reasoning as https://github.com/Winetricks/winetricks/pull/1338/files/7c0eed4e8a25108c4f9e345657356343a07cbf78#r332377677 for windows_windows

windows_cygwin used in previous and required to avoid attempting to run WINE.